superset-frontend/src/explore/components/DatasourcePanel/index.tsx

```diff
@@ -190,6 +217,35 @@ export default function DataSourcePanel({
     [_columns],
   );
 
+  const placeholderSlDataset = {
+    sl_table: [],
```

Comment:
why are we using arrays as placeholders here?

Reply:
I wasn't sure what these properties were going to be but thought I needed to pass in something as a placeholder until the back end is implemented. Turns out I don't need it, so I removed it in [`this commit`](https://github.com/apache/superset/pull/19855/commits/fd65bfa6f6241c466fa4eac90efafcc87f5c95cd). Thanks for this catch!